bundle/manifests/compliance-operator.clusterserviceversion.yaml

```diff
@@ -206,6 +206,11 @@ spec:
       kind: ComplianceSuite
       name: compliancesuites.compliance.openshift.io
       version: v1alpha1
+    - description: CustomRule is the Schema for the customrules API
```

Comment:
I see this is similar to what we have for `Rule`, but isn't not overly descriptive about what it does or why someone would want to use it.

Perhaps we can update it to something more like the ComplianceScan or ComplianceRemediation CRDs?

```yaml
- description: CustomRule represents a rule that can be used with TailoredProfiles to execute arbitrary checks against the cluster.
```